test/unit/dca-hub-position-descriptor-fixed.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { utils } from 'ethers';
+import { ethers, network } from 'hardhat';
+import { IDCAPositionGetter, DCAHubPositionDescriptorFixed, DCAHubPositionDescriptorFixed__factory, IERC20Metadata } from '@typechained';
+import chai, { expect } from 'chai';
+import { FakeContract, smock } from '@defi-wonderland/smock';
+import { given, then, when } from '@utils/bdd';
+
+chai.use(smock.matchers);
+
+describe('DCAHubPositionDescriptorFixed', () => {
+  let hub: FakeContract<IDCAPositionGetter>;
+  let tokenId: number;
+  let DCAHubPositionDescriptorFixed: DCAHubPositionDescriptorFixed;
+
+  before('Setup accounts and contracts', async () => {
+    hub = await smock.fake('IDCAPositionGetter');
+    tokenId = Math.floor(Math.random() * (10000 - 1));
+    const factory = (await ethers.getContractFactory('DCAHubPositionDescriptorFixed')) as DCAHubPositionDescriptorFixed__factory;
+    const [signer] = await ethers.getSigners();
+    DCAHubPositionDescriptorFixed = await factory.deploy(signer.address);
+  });
```

Comment:
You can read the chain id on the `before` with `(await ethers.provider.getNetwork()).chainId`